### ISSUE TITLE
"Formal Pants" Incorrect Attribution (question)

### ISSUE THREAD

**Issue by jrconway3** [sanderfrenken/Universal-LPC-Spritesheet-Character-Generator#228]
This is sort of a question, I could easily make the change myself.

I did some research to figure out WTF the "formal" pants came from. I also found a "furbrown" pants as well.

Anyway, this is a strange one. I thought that Wulax's pants were the OG pants. Technically speaking, they were, in a way. However, Stephen Challener (Redshrike) also made his own pants. These pants largely got forgotten, even though they are still present in the repo. They never got proper recolors, though, and kept getting mixed in with other pants assets.

I think Bluecarrot16 mistakenly credited these to Laetissima and Pennomi:
https://opengameart.org/content/lpc-2-characters

Because the pants were used in here. However, at no point did they take credit for the pants, only for the collared shirt, bowtie, and vest. The pants were just the base pants by Redshrike.

Bluecarrot did use other assets from that, but the Pants in the repo are incorrectly attributed to Laetissma and Pennomi as far as I can tell.

I'm thinking this should be something that I could just fix, but I don't want to jump to making a change like this without permission.

Bluecarrot probably did clean up these pants a bit, and the "striped" variant is probably also Bluecarrot's (because that one isn't part of the base assets).

**Comment by jrconway3** [comment]
Similarly, I don't see any evidence Wulax contributed to "Pantaloons". These are simply Nilla122's pants:
https://opengameart.org/content/more-lpc-clothes-and-hair

**Comment by sanderfrenken** [comment]
Oef I also don't know about this, maybe Bluecarrot can clarify it to some extent, did you check with him?

**Comment by jrconway3** [comment]
Yeah I haven't been in contact much with Bluecarrot. Bluecarrot did leave some comments on my recent submissions on OGA, but that's about it so far.

I think these are the correct credits based on what I'm seeing, and previously I would normally just fix it, but I'm trying to be a bit more cautious about changing the credits now.

**Comment by jrconway3** [comment]
I think if I see more credits-related issues I'll drop them in here, too.

**Comment by jrconway3** [comment]
So just to clarify, here's the breakdown of "Formal Pants" in the credits. The following are credited:

bluecarrot16, Thane Brimhall (pennomi), laetissima, Johannes Sjölund (wulax)

The licenses are credited:
CC-BY-SA 3.0, GPL 3.0

And these links are provided::
https://opengameart.org/content/liberated-pixel-cup-lpc-base-assets-sprites-map-tiles
https://opengameart.org/content/lpc-medieval-fantasy-character-sprites
https://opengameart.org/content/lpc-2-characters
https://opengameart.org/content/lpc-gentleman

So Formal Pants are based on LPC Gentleman, which were based on the pants that were included in "LPC 2 Characters" Again, though, those pants are just identical to the ones by Redshrike in the base assets.

Now, I don't know why Wulax is credited here or his Medieval Fantasy Character Sprites is listed, unless Bluecarrot took some elements from Wulax's pants variant? As far as I can tell, though, Bluecarrot16's pants are just taken from the LPC 2 Characters (which, again, Pennomi and Laetissima did not contribute, nor do they take credit for; they just had them in the preview and XCF as they used the base pants by Redshrike).

LPC Gentlemen doesn't credit Wulax either:
https://opengameart.org/content/lpc-gentleman

So I think we can at least safely drop Wulax from this particular credit.


That said, my understanding is these pants should also include OGA-BY 3.0 and it should only be Bluecarrot16 and Redshrike (until I make more frames with these pants, in any case :P).

**Comment by sanderfrenken** [comment]
Shall we merge this issue with https://github.com/sanderfrenken/Universal-LPC-Spritesheet-Character-Generator/issues/142 as both of them are is about (incorrect/ incomplete) licensing?

**Comment by jrconway3** [comment]
I suppose we could just have one card to point out specific licensing issues.
